src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -452,13 +452,15 @@ def increment_timeout(self, increment):
                 "Incrementing %s from %s to %s seconds", section, value, value + increment)
             set_avocado_config_value(namespace, key, value + increment)
 
-    def log_step(self, message):
+    def log_step(self, message, marker=False):
         """Log a test step.
 
         Args:
             message (str): description of test step.
-
+            marker (bool, optional): whether to log a marker line in the log. Defaults to False.
         """
+        if marker:
+            self.log.info('-' * 80)
```

Comment:
Maybe a better name is "separator" or "header"? "marker" makes me think it will be on the same line, or that it will be some special character

Reply:
Done, thanks.